index.bs

```diff
@@ -1660,6 +1334,94 @@ multiple transactions per microframe. In a SuperSpeed device this value will
 include the multiplication factor provided by the <code>bMaxBurst</code> field
 of the SuperSpeed Endpoint Companion descriptor.
 
+# Integrations # {#integrations}
+
+## Feature Policy ## {#feature-policy}
+
+This specification defines a <a>feature</a> that controls whether the
+{{Navigator/usb}} attribute is exposed on the {{Navigator}} object.
```

Comment:
For the pre-existing permission-based features being controlled by policy, we've generally tried to suggest disabling them by rejecting permission/usage requests, rather than restricting attribute exposure.

If exposure is the best fit for a new API, then we can definitely do that (We'll have to reinstate the IDL section in the Feature Policy spec) -- I don't think that FP should be prescribing the exact mechanism for any particular feature. There's something to be said for consistency within the overall platform, though.